tools/aw2db.js

```diff
@@ -347,7 +380,7 @@ function* parsePropFile(path) {
  */
 function fixEncoding(content) {
   // Fix faulty line breaks
-  return content.replaceAll('€\u007F', '\r\n');
+  return content.replaceAll('€\u007F', '\r\n').replaceAll('�\u007F', '\r\n');
```

Comment:
For the second `replaceAll` statement I see "�" displayed: is it specifically this you want to catch or is it the result of the code preview not being able to print the actual character?
In case it's the latter, then use the escaped Unicode sequence (like `\u007F` is) corresponding to the actual character.

Reply:
Yes, that is indeed the sequence of characters that shows up in the db and not a bug with your browser / the code preview.